packages/editor/src/components/properties/ImageNodeEditor.tsx

```diff
@@ -45,19 +45,30 @@ export const ImageNodeEditor: EditorComponentType = (props) => {
   const imageComponent = useComponent(entity, ImageComponent)
   const errors = getEntityErrors(props.entity, ImageComponent)
 
+  const imageSource =
```

Comment:
These properties have been removed recently, please make sure you are working off the latest dev version. We just need image.source here

Reply:
![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/02c95f82-0255-4ab6-a2a6-6bfe50276797)

Keeping only 2 options: `imageComponent.resource.value?.source || imageComponent.source.value`